assets/css/footer.css

```diff
@@ -31,10 +31,7 @@ footer > .footer-row {
   text-align: left;
 }
 
-.footer-column .sub-footer-link {
-  display: flex;
-  flex-wrap: wrap;
-}
```

Comment:
Why did you remove this styling?

Reply:
The items that are listed in group 1 is arranged in a horizontal manner,3 items in first row,1 item in second row. I thought it would look nice when we  arrange those items one by one.

Can I change the display to block?